config/initializers/twitter_credentials.rb

```diff
@@ -0,0 +1,6 @@
+$twitter = Twitter::REST::Client.new do |config|
```

Comment:
Be very careful about using global variables. It can get messy. Many Rubyists advise against using at all.